lib/money/display_options.ex

```diff
@@ -31,8 +31,24 @@ defmodule Money.DisplayOptions do
   @enforce_keys @all_fields
   defstruct @all_fields
 
+  defp custom_display_options do
```

Comment:
```suggestion
  @spec custom_display_options() :: Keyword.t()
  defp custom_display_options do
```

In addition, I would probably move this function to the end of the file.

Reply:
same here, this is not a `Keyword.t()` , this is a `map()`